en/developer_guide/external_media_domain.rst

```diff
@@ -0,0 +1,26 @@
+Adding an external media domain
+===============================
+
+Configuration
+~~~~~~~~~~~~~
+
+Sometimes, developers want an external media domain to improve websites performances.
+Open Orchestra allow to add a domain really easily in configuration:
+
+.. code-block:: yaml
+
+    open_orchestra_media:
+        media_domain: http://exemple.com
+
+Create your external media domain
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To be compatible, your server need to install Symfony with `open-orchestra-media-bundle`_.
+Then `configure gaufrette`_ to choose your stocking parameters.
+An exemple is avaible to `open-orchestra-media-demo`_
+
+That's all !
```

Comment:
Or you should do it a magnificent way :p
![That's all](http://www.blacktriangle.org/blog/images/folks.jpg)